ocs_ci/utility/lvmo_utils.py

```diff
@@ -0,0 +1,145 @@
+import logging
```

Comment:
Is there a reason not to put this in the LVM class?

Reply:
Followed [this](https://github.com/red-hat-storage/ocs-ci/blob/5192d7601f9e8bc709c70b3dedba09afa97d31c3/ocs_ci/utility/utils.py#L2038) convention